spacy/pipeline/coref.py

```diff
@@ -511,15 +509,20 @@ def update(
         set_dropout_rate(self.model, drop)
 
         total_loss = 0
-
-        for eg in examples:
-            preds, backprop = self.model.begin_update([eg.predicted])
-            score_matrix, mention_idx = preds
-
-            loss, d_scores = self.get_loss([eg], score_matrix, mention_idx)
+        docs = [eg.predicted for eg in examples]
+        for doc, eg in zip(docs, examples):
+            # replicates the EntityLinker's behaviour and
+            # copies annotations over https://bit.ly/3iweDcW
```

Comment:
Even if it's long enough it needs its own line, I would avoid using a bitly link here - just use the direct URL.